### DISCUSSION TITLE
Port Timeout

### DISCUSSION THREAD

**Discussion by manerjeff** [blinksh/blink#1863] [Support]
In build I am trying to expose ports using: **build ports add htools 5000 5001**. I am receiving an error:

build ports add htools 5000 5001
WgHttpError(
    RequestTimeout(
        4s,
    ),
)
blink> 

I am running on both an iPad Pro and an iPad mini. Please note I am also running the developer beta of iPad OS 17, I do not know if that is related. Thanks

**Comment by carloscabanero** [comment]
Hi! That usually means that the remote Build machine could not be reached. Can you check that you can do `build mosh` and that `build machine status` returns all the information?
